poi-scratchpad/bin/.gitignore

```diff
@@ -0,0 +1,2 @@
+/main/
+/test/
```

Comment:
all these gitignore changes should be removed - you can update your own user .gitignore but we don't need these in svn - POI is not a Git project - it is an svn project